src/groovy/heim/session/SessionService.groovy

```diff
@@ -24,6 +24,16 @@ import java.util.concurrent.ConcurrentMap
 @Log4j
 class SessionService {
 
+
+    def SessionService() {
+        new Timer().schedule({
+            garbageCollection()
+        } as TimerTask, collectionInterval, collectionInterval)
+    }
+
+    private final Integer collectionInterval = 3 * 60 * 1000 // 3 min
+    private final Integer sessionLifeSpan = 10 * 60 * 1000 // 10 min
```

Comment:
Move fields before methods and constructors; constants are written with capitals.